src/main/java/Event.java

```diff
@@ -0,0 +1,28 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalTime;
+
+public class Event extends Task {
+
+    protected String dateTime;
+    protected String date;
+    protected String time;
+    protected LocalDate date1;
+    protected LocalTime time1;
```

Comment:
Perhaps you might want to use a better name for `date1` and `time1`?

Reply:
Thanks for the advice !